src/Stratis.Bitcoin/Consensus/CoinViews/ICachedCoinView.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using NBitcoin;
+using Stratis.Bitcoin.Utilities;
+
+namespace Stratis.Bitcoin.Consensus.CoinViews
+{
+    /// <summary>
+    /// Database of UTXOs.
```

Comment:
What does 'Database of UTXOs' mean in terms of this interface ICachedCoinView ?

Reply:
I haven't changed the summary, it was like this before. It only shows as new file because I moved it to another project.